foundation/foundation-mda/src/main/resources/templates/deployment/airflow/v2/airflow.chart.yaml.vm

```diff
@@ -31,5 +31,5 @@ appVersion: "1.0.0"
 dependencies:
   - name: aissemble-airflow-chart
     version: ${versionTag}
```

Comment:
I: these were preventing testing this change and needed to be fixed anyway - so here we go